fact_extractor/install/unpacker.py

```diff
@@ -1,19 +1,16 @@
+import hashlib
 import logging
 import os
 from getpass import getuser
 from pathlib import Path
+from shlex import split
+from subprocess import CalledProcessError, run
 from tempfile import TemporaryDirectory
 
 from common_helper_process import execute_shell_command_get_return_code
 
-from helperFunctions.install import (
-    apt_install_packages,
-    install_github_project,
-    InstallationError,
-    OperateInDirectory,
-    pip_install_packages,
-    apt_remove_packages,
-)
+from helperFunctions.install import (apt_install_packages, apt_remove_packages, install_github_project,
+                                     InstallationError, OperateInDirectory, pip_install_packages)
```

Comment:
style: I think the way it was before was more beautiful.

Reply:
Yes, it's because of the missing configuration for isort/black. I will add it so that it matches the configuration in FACT_core